### DISCUSSION TITLE
Is it necessary to store data when the pages change?

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#5596] [General]
When navigating between two pages, the storage manager currently triggers an API call each time. This is unnecessary. We should consider storing the data to avoid making continuous API calls during page changes. Imagine a user frequently switching between pages, leading to constant API hits. This could become a serious issue.

**Comment by RakulAgn** [comment]
@artf

**Comment by artf** [comment]
Are you able to create a reproducible demo? From a quick test, switching pages doesn't trigger any storage action on my side

**Reply by RakulAgn** [reply]
https://github.com/GrapesJS/grapesjs/assets/119593989/0c6c512c-b15e-4fcc-8cac-33feab1b8ea0

![image](https://github.com/GrapesJS/grapesjs/assets/119593989/78dd4880-1d58-4bb7-ad94-cad27dc866b9)
![image](https://github.com/GrapesJS/grapesjs/assets/119593989/c67524a2-34ac-4084-9ed4-1f226a3f4d5e)
![image](https://github.com/GrapesJS/grapesjs/assets/119593989/5b6e302a-9f8a-4d85-99e3-5be289135581)

This is what i am doing and configured

**Reply by RakulAgn** [reply]
@artf

**Reply by artf** [reply]
> This is what i am doing and configured

I didn't ask to see what you're doing or what is happening (I believe you), I asked for a minimal reproducible demo so we're both able to see there are no storage events/actions on the page switch.
